rmgpy/kinetics/arrhenius.pyx

```diff
@@ -249,7 +249,23 @@ cdef class Arrhenius(KineticsModel):
 
         # Set the rate parameter to a cantera Arrhenius object
         ctReaction.rate = self.toCanteraKinetics()
-    
+
+    cpdef ArrheniusEP toArrheniusEP(self):
```

Comment:
I added a method to allow the alpha to be changed, as some people might want to use it (though in my experience, it is unlikely to occur before the code is widely rewritten). I also added test methods for it as well. 

I agree with @rwest that deltaH would **need** to be passed so there is an exception if only alpha or deltaH is set. If only alpha is passed the ArrheniusEP would be wrong and the user would get incorrect values.